package/cfg/rke-cis-1.23-permissive/etcd.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 controls:
-version: "cis-1.23"
+version: 1.23
```

Comment:
@KiranBodipi regarding the version name, I agree we should stick to kube-bench's format which uses `version: 1.X `and not `version: cis-1.x`.
IMO the cis-benchmark chart (or any other components) use cfgs filenames, so setting back the naming convention to version: 1.X in the yamls should be fine.
@rayandas could you confirm this ?
If we all approve this, let's change all files to use `version: 1.X`.

Reply:
Yeah we normally follow `version: 1.x` only. But I saw for a few configs there are `version: cis-1.x` which we can change to `version: 1.x`

Reply:
Or we can keep all the configs as `version: cis-1.x` same as kube-bench. I shouldn’t impact anything.

Reply:
Raised https://github.com/rancher/cis-operator/issues/248 to track the changes. Someone from my team will rake it up.